applications/DEMApplication/python_scripts/DEM_procedures.py

```diff
@@ -974,16 +974,16 @@ def evaluate_computation_of_fem_results():
         evaluate_computation_of_fem_results()
 
     def MoveAllMeshes(self, all_model_parts, time, dt): # TODO: deprecated
+        message = 'Warning!'
+        message += '\nFunction \'MoveAllMeshes\' is deprecated. It is called inside sphere_strategy.py'
+        message += '\nIt will be removed after 10/31/2019.\n'
+        Logger.PrintWarning("DEM_procedures.py", message)
 
         spheres_model_part = all_model_parts.Get("SpheresPart")
         dem_inlet_model_part = all_model_parts.Get("DEMInletPart")
-        rigid_face_model_part = all_model_parts.Get("RigidFacePart")
-        cluster_model_part = all_model_parts.Get("ClusterPart")
 
-        self.mesh_motion.MoveAllMeshes(rigid_face_model_part, time, dt)
```

Comment:
just out of curiosity, why are you removing the rigid face model part from the movements?

Reply:
It is done here:
https://github.com/KratosMultiphysics/Kratos/blob/6f91aeb60e3d5d665dacecc1e0c31ff232e209b9/applications/DEMApplication/custom_strategies/strategies/explicit_solver_strategy.cpp#L688
So we were doing it twice per time step (bug alert!).
We think that the strategy and scheme must take care of moving the walls. Depending on the combination of strategy and scheme, both the spheres and the walls should be moved in one step, or in two steps, or predicting and correcting iteratively, etc. Eventually, the MoveAllMeshes function should disappear, we are migrating it step by step.

Reply:
I forgot to say that all walls (understanding a wall as a set of triangular or quadrilateral conditions) are converted into RigidBody elements at Initialize. If they are fixed, no motion is applied to them.

Reply:
Ok, thanks! 
I see your point. It seems that this is quite a complex topic within DEM.

Reply:
@tteschemacher  It seems like I was totally wrong. That line only moves the walls if they form a RigidBody which can move freely, or with some restricted dofs. If the walls are totally fixed then they must be moved by MoveAllMeshes.
When you see this strange mixes we are doing, you realize how importat migrating to processes is. However, they have a problem: processes can't get into the Strategy or the scheme, they can only be called by the AnalysisStage at InitializeSolutionStep, FinalizeSolutionStep and so on...